torchx/schedulers/ray/ray_driver.py

```diff
@@ -130,24 +139,43 @@ def main() -> None:  # pragma: no cover
     actors: List[RayActor] = load_actor_json("actors.json")
     # pyre-fixme[16]: Module `worker` has no attribute `init`.
     ray.init(address="auto", namespace="torchx-ray")
-    pg: PlacementGroup = create_placement_group(actors)
-    command_actors: List[CommandActor] = create_command_actors(actors, pg)
 
-    active_workers = [
-        command_actor.exec_module.remote()  # pyre-ignore
-        for command_actor in command_actors
-    ]
+    pending_placement_groups = [
+        create_placement_group(actors[i : i + 1]) for i in range(len(actors))
```

Comment:
this should request min replicas first right instead of a one size pg for each replica?

Reply:
what @d4l3k said, but also I think you have created this method `create_placement_group_async` but are never calling it. your code is still calling the `create_placement_group` in the main function

Reply:
since we are inrementally creating the placement group, do we/should we even deal with the fact that if a pg creation fails we just get cancel the creation of the rest of the pgs, or do we just take that as a separate task

Reply:
It's old code, in the latest code, it's async.

Reply:
is the new code on a different branch? should it be in this PR os a new PR. 

also other alternative might be the take these 2 loops and put them in two different actors that are basically running these loops in parallel.